src/pages/storey.mdx

```diff
@@ -3,3 +3,24 @@ tags: ["storey"]
 ---
 
 # Introduction
+
+[Storey](https://docs.rs/cw-storey/latest/cw_storey/) is an alternative to
+[StoragePlus](cw-storage-plus) and fulfills the same purpose. The
+[StoragePlus introduction](cw-storage-plus) could very well be transplanted
+here, verbatim - feel free to give it a read if you're not sure what this
+software does.
+
+While StoragePlus has been around for years and is the established way of
+storing your data on-chain, Storey is the new kid on the block.
+
+Storey is currently **prototype software**, meaning the API might change
+rapidly, **storage layout might break between versions**, and you might stumble
+into bugs and missing features. It's probably a good idea not to use it for
+serious production software just yet. If you do, please keep in mind the
+increased maintenance burden and be careful!
```

Comment:
Should this maybe be in a call-out block to make it more visually distinct?

Reply:
Yeah, good call